WalletWasabi/Nito/AsyncEx/IdManager.cs

```diff
@@ -24,27 +24,23 @@ internal static class IdManager<TTag>
 		/// <param name="id">A reference to the field containing the id.</param>
 		public static int GetId(ref int id)
 		{
-			// If the Id has already been assigned, just use it.
-			if (id != 0)
+			if (id == 0)
```

Comment:
This is not clearer than the previous version and adds an extra level of identation.